src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
```

Comment:
I'm not really that scared of polyfills in this case. In fact, I wouldn't
mind checking to see that it works with Babel.

On Tue, Jun 2, 2015 at 3:31 PM Arthur Verschaeve <notifications@github.com>
wrote:

> In src/core.js
> <https://github.com/jquery/jquery/pull/2369#discussion_r31561468>:
>
> > @@ -425,6 +425,16 @@ jQuery.extend({
> >  	support: support
> >  });
> >
> > +// jsHint would error on this code due to the Symbol not being defined in ES5.
> > +// Defining this global in .jshintrc would create a danger of using the global
> > +// unguarded in another place, it seems safer to just disable jsHint for these
>
>  jsHint
>
> I always thought the correct way to capitalize it is JSHint.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/jquery/pull/2369/files#r31561468>.
>